test/if-then-else.test.js

```diff
@@ -391,8 +391,8 @@ t.test('if/else with const integers', (t) => {
   t.plan(2)
 
   const schema = {
-    type: 'integer',
```

Comment:
`integer` should be a valid `type`. Why change it to `number`?

ttps://json-schema.org/understanding-json-schema/reference/numeric.html#id4

Reply:
test/if-then-else.test.js 2> strict mode: type "number" not allowed by context "integer" at "9f255981-9e81-41ed-8a47-ec359679ccd7#/if" (strictTypes)

Reply:
@climba03003 
Does this answer your question?

Reply:
It does answer my question, but I would like to hear why it is added like this in the beginning.
This test is added in https://github.com/fastify/fast-json-stringify/pull/533

cc @ivan-tymoshenko

Reply:
> I would like to hear why it is added like this in the beginning

It made some sense before this change. https://github.com/fastify/fast-json-stringify/pull/533#discussion_r967678324

It's ok to change it now.